internal/infrastructure/kubernetes/ratelimit/resource.go

```diff
@@ -34,6 +36,17 @@ const (
 	InfraName = "envoy-ratelimit"
 	// InfraGRPCPort is the grpc port that the rate limit service listens on.
 	InfraGRPCPort = 8081
+	// ConfigType is the configuration loading method for ratelimit.
+	ConfigType = "CONFIG_TYPE"
+	// ConfigGrpcXdsServerURL is the url of ratelimit config xds server.
+	ConfigGrpcXdsServerURL = "CONFIG_GRPC_XDS_SERVER_URL"
+	// ConfigGrpcXdsNodeID is the id of ratelimit node.
+	ConfigGrpcXdsNodeID = "CONFIG_GRPC_XDS_NODE_ID"
+
+	// XdsGrpcSotwConfigServerPort is the listening port of the RateLimit xDS config server.
+	XdsGrpcSotwConfigServerPort = 18001
```

Comment:
I haven't expose this port in envoy-gateway service yet. see if is ok to open this port for xds config server? @arkodg

Reply:
sure since 
```
DefaultXdsServerPort = 18000
```

Reply:
okay, sure